ios/Demo-iOS/Sources/Views/AppRootView.swift

```diff
@@ -40,9 +40,13 @@ struct AppRootView: View {
         }
         .onChange(of: self.selectedConfiguration) { oldValue, newValue in
             switch newValue {
-            case .bundledEditor: activeEditorConfiguration = createBundledConfiguration()
-            case .editorConfiguration(let config): self.loadEditorConfiguration(for: config)
-            case .none: self.activeEditorConfiguration = nil
+            case .bundledEditor:
+                let config = createBundledConfiguration()
+                activeEditorConfiguration = config
```

Comment:
Likely worthwhile consistently using `self` in this `switch`.
```suggestion
                self.activeEditorConfiguration = config
```